lib/manageiq/rpm_build/generate_gemset.rb

```diff
@@ -69,6 +69,9 @@ def populate_gem_home(build_type)
             shell_cmd("gem install unf_ext -v '0.0.7.2' -- --with-cxxflags='-fsigned-char'")
           end
 
+          shell_cmd("if [ $(uname -m) = s390x ]; then gem install sassc  -- --disable-march-tune-native; fi")
+          shell_cmd("if [ $(uname -m) = s390x ]; then gem install unf_ext -v '0.0.7.2' -- --with-cxxflags='-fsigned-char'; fi")
+
```

Comment:
The commands look identical to powerpc64le, please combine into the if statement above.